Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -0,0 +1,301 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+'''IMPORTS'''
+import requests
+from datetime import datetime
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBAL VARS '''
+BASE_URL = demisto.getParam('host').rstrip('/') + '/v1.0/'
+TENANT = demisto.getParam('tenant')
+TOKEN = demisto.getParam('token')
+HEADERS = {"Authorization": TOKEN, "Accept": "application/json"}
+USE_SSL = not demisto.params().get('unsecure', False)
+
+''' CONSTANTS '''
+# DEMISTOBOT = "https://demistobot.demisto.com/msg-user" todo: update when ready
+DEMISTOBOT = 'https://ec2-18-197-54-7.eu-central-1.compute.amazonaws.com/msg-user-token/'
+PRODUCT = "MicrosoftGraphUser"
+BLOCK_ACCOUNT_JSON = '{"accountEnabled": false}'
+UNBLOCK_ACCOUNT_JSON = '{"accountEnabled": true}'
+NO_OUTPUTS = {}
+
+
+def camel_case_to_readable(text):
+    """
+    'camelCase' -> 'Camel Case'
+    """
+    if text == 'id':
+        return 'ID'
+    return ''.join(' ' + char if char.isupper() else char.strip() for char in text).strip().title()
+
+
+def parse_outputs(users_data):
+    """
+    Parse user data as received from Microsoft Graph API into Demisto's conventions
+    """
+    if isinstance(users_data, list):
+        users_readable, users_outputs = [], []
+        for user_data in users_data:
+            user_readable = {camel_case_to_readable(k): v for k, v in user_data.items() if k != '@removed'}
+            if '@removed' in user_data:
+                user_readable['Status'] = 'deleted'
+            users_readable.append(user_readable)
+            users_outputs.append({k.replace(' ', ''): v for k, v in user_readable.copy().items()})
+
+        return users_readable, users_outputs
+
+    else:
+        user_readable = {camel_case_to_readable(k): v for k, v in users_data.items() if k != '@removed'}
+        if '@removed' in users_data:
+            user_readable['Status'] = 'deleted'
+        user_outputs = {k.replace(' ', ''): v for k, v in user_readable.copy().items()}
+
+        return user_readable, user_outputs
+
+
+def epoch_seconds(d=None):
+    """
+    Return the number of seconds for given date. If no date, return current.
+    """
+    if not d:
+        d = datetime.utcnow()
+    return int((d - datetime.utcfromtimestamp(0)).total_seconds())
+
+
+def get_token():
```

Comment:
If I understand correctly, In case token was revoked before time, we will get error until the round hour will pass.
maybe we can add `get_token` an argument to discard the old token anyway, than catch the relevant response and call it then.
also, what is the token life time? is it configurable?